add ilist dataset (#1505)
huggingface/datasets: datasets/ilist/README.md

---
task_categories:
- text-classification
multilinguality:
- multilingual
task_ids:

> - other

Reviewer: `multi-class-classification` maybe ? @lhoestq WDYT ?

Reviewer 2: I would say language identification.
You can use
```
task_ids:
- text-classification-other-language-identification
- ```

    - [Licensing Information](#licensing-information)
    - [Citation Information](#citation-information)

## Dataset Description

- **Homepage:**
- **Repository:** [GitHub](https://github.com/kmi-linguistics/vardial2018)
- **Paper:** [Link](https://www.aclweb.org/anthology/W18-3900/)
- **Leaderboard:**
- **Point of Contact:** linguistics.kmi@gmail.com

### Dataset Summary

This datasets is introduced in a task which aimed at identifying 5 closely-related languages of Indo-Aryan language family – Hindi (also known as Khari Boli), Braj Bhasha, Awadhi, Bhojpuri and Magahi. These languages form part of a continuum starting from Western Uttar Pradesh (Hindi and Braj Bhasha) to Eastern Uttar Pradesh (Awadhi and Bhojpuri) and the neighbouring Eastern state of Bihar (Bhojpuri and Magahi). For this task, participants were provided with a dataset of approximately 15,000 sentences in each language, mainly from the domain of literature, published over the web as well as in print.

### Supported Tasks and Leaderboards

[More Information Needed]

### Languages

 Hindi, Braj Bhasha, Awadhi, Bhojpuri and Magahi

## Dataset Structure

### Data Instances

> [More Information Needed]

Reviewer: It would be great to have a sample here, in json format. Something like:
```json
{
    "language_id": "hi",
    "text": "some text in hindi"
}
```

### Data Fields

> - `language_id`: a `string`

Reviewer: Can you be more explicit on what this string is? Is it the language code?

- `text`: text which you want to classfiy

### Data Splits

> |                            | Train   | Valid | Test |
> | -----                      | ------ | ----- | ---- |
> | Input Sentences            |   70351     |   9692    |  10329    |

Reviewer: Just some aestethics 😇

```suggestion
|                      | train | valid | test  | 
|----------------------|-------|-------|-------|
| # of input sentences | 70351 | 9692  | 10329 |
```

    - [Licensing Information](#licensing-information)
    - [Citation Information](#citation-information)

## Dataset Description

- **Homepage:**
- **Repository:** [GitHub](https://github.com/kmi-linguistics/vardial2018)
- **Paper:** [Link](https://www.aclweb.org/anthology/W18-3900/)
- **Leaderboard:**
- **Point of Contact:** linguistics.kmi@gmail.com

### Dataset Summary

This datasets is introduced in a task which aimed at identifying 5 closely-related languages of Indo-Aryan language family – Hindi (also known as Khari Boli), Braj Bhasha, Awadhi, Bhojpuri and Magahi. These languages form part of a continuum starting from Western Uttar Pradesh (Hindi and Braj Bhasha) to Eastern Uttar Pradesh (Awadhi and Bhojpuri) and the neighbouring Eastern state of Bihar (Bhojpuri and Magahi). For this task, participants were provided with a dataset of approximately 15,000 sentences in each language, mainly from the domain of literature, published over the web as well as in print.

### Supported Tasks and Leaderboards

[More Information Needed]

### Languages

 Hindi, Braj Bhasha, Awadhi, Bhojpuri and Magahi

## Dataset Structure

### Data Instances

> {
>     "language_id": 4,
>     "text": 'तभी बारिश हुई थी जिसका गीलापन इन मूर्तियों को इन तस्वीरों में एक अलग रूप देता है .'
> }

Reviewer 2: ````suggestion
```
{
    "language_id": 4,
    "text": 'तभी बारिश हुई थी जिसका गीलापन इन मूर्तियों को इन तस्वीरों में एक अलग रूप देता है .'
}
```
````

### Data Fields

- `text`: text which you want to classfiy

> - `language_id`: label for the text

Reviewer 2: ```suggestion
- `language_id`: label for the text as an integer from 0 to 4

The language ids correspond to the following languages: "AWA", "BRA", "MAG", "BHO", "HIN"
```

## Dataset Description

- **Homepage:**
- **Repository:** [GitHub](https://github.com/kmi-linguistics/vardial2018)
- **Paper:** [Link](https://www.aclweb.org/anthology/W18-3900/)
- **Leaderboard:**
- **Point of Contact:** linguistics.kmi@gmail.com

### Dataset Summary

This datasets is introduced in a task which aimed at identifying 5 closely-related languages of Indo-Aryan language family – Hindi (also known as Khari Boli), Braj Bhasha, Awadhi, Bhojpuri and Magahi. These languages form part of a continuum starting from Western Uttar Pradesh (Hindi and Braj Bhasha) to Eastern Uttar Pradesh (Awadhi and Bhojpuri) and the neighbouring Eastern state of Bihar (Bhojpuri and Magahi). For this task, participants were provided with a dataset of approximately 15,000 sentences in each language, mainly from the domain of literature, published over the web as well as in print.

### Supported Tasks and Leaderboards

[More Information Needed]

### Languages

 Hindi, Braj Bhasha, Awadhi, Bhojpuri and Magahi

## Dataset Structure

### Data Instances

```
{
    "language_id": 4,
    "text": 'तभी बारिश हुई थी जिसका गीलापन इन मूर्तियों को इन तस्वीरों में एक अलग रूप देता है .'
}
```

### Data Fields

> - `text`: text which you want to classfiy
> - `language_id`: label for the text

Reviewer 2: ```suggestion
- `text`: text which you want to classify
- `language_id`: label for the text as an integer from 0 to 4
The language ids correspond to the following languages: "AWA", "BRA", "MAG", "BHO", "HIN".
```
